### DISCUSSION TITLE
Is Sunbird allows moving the course from one organization to another?

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Knowlg/Community#132] [General Discussions]
Hello @Krishnaj20 @pallakartheekreddy,

Could you please assist us with the following question?

**Question** - Is Sunbird allows moving the course from one organization to another? If yes, then can you please provide the steps for the same?
**Sunbird Version** - Release-4.4.1

cc @Renuka-S

**Comment by pallakartheekreddy** [comment]
@komalm in Sunbird Knowlg, we don't have move option to move the course from one organization to another. But, We can copy the course from one organization to another organization using the copy API. Please [ref](https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service-1/content-api) here for API details

**Reply by pallakartheekreddy** [reply]
@komalm Hope your query is now resolved. Please mark the thread as answered
